test/README.md

```diff
@@ -17,7 +17,8 @@ To run locally all you need is:
 1. A POSIX shell
 1. `rustup target add x86_64-unknown-linux-musl`
 1. `cargo install bpf-linker`
-1. Install `qemu` and `cloud-init-utils` package - or any package that provides `cloud-localds`
+1. `go install github.com/florianl/bluebox@latest`
```

Comment:
I don't love that we need a go toolchain now :(
Is there a way we can use binary installs?

In fact even better would be for `xtask` to manage the version installed in somewhere like `xtask/bin`